ibm/ibm-how-to/launch-ubuntu-instances.rst

```diff
@@ -1,125 +1,107 @@
 
 
 Choose a zone and region
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-When creating a VSI, you'll need to specify a region and zone for the VSI to be created in.
+When creating a VSI, you'll need to specify a region and zone for the VSI to be created in. For a list of available regions and zones, refer to `IBM's documentation about regions`_.
 
-For the purposes of this how-to, we'll use the `us-south` region and the `us-south-1` zone. You can choose a different
-region and zone if you prefer.  
-Refer to the official IBM Cloud documentation for a list of available regions and zones: https://cloud.ibm.com/docs/vpc?topic=vpc-creating-a-vpc-in-a-different-region&interface=cli
 
-
-Set the region:
+Set the region (e.g. us-south):
 
 .. code-block:: bash
 
    ibmcloud is target --region us-south
 
 
-Create all other resources needed for the VSI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Refer to IBM Cloud's 'Creating VPC resources with CLI and API' documentation for instructions on creating a VPC, subnet,
-and security group, if you do not already have these resources set up:
-https://cloud.ibm.com/docs/vpc?topic=vpc-creating-vpc-resources-with-cli-and-api&interface=cli#creating-a-vpc-using-cli
+Create other needed resources
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Be sure to note the ID of the VPC you wish to launch the VSI in. 
+If you don't have an existing VPC, subnet and security group, set them up by following `IBM's instructions for creating VPC resources`_. Note the created VPC ID for later use.
 
-For this how-to, we'll just use an existing VPC. To list existing VPCs, run:
+For existing VPCs, you can list them using:
 
 .. code-block:: bash
 
    ibmcloud is vpcs
 
 
-Launch the Virtual Server Instance
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Launch the VSI
+~~~~~~~~~~~~~~
 
-With all necessary information in hand, create your VSI:
+Create the VSI using:
 
 .. code-block:: bash
 
-   ibmcloud is instance-create MyInstance <vpc-id> <zone> <instance-type> <image-id> --keys <ssh-key-id>
-
+   ibmcloud is instance-create MyInstance \
+            <vpc-id> <zone> <instance-type> <image-id> \
+            --keys <ssh-key-id>
 
-- Replace `<ssh-key-id>` with the ID of the SSH key you wish to login to the VSI with.
-- Replace `<vpc-id>` with the ID of the VPC you wish to launch the VSI in.
-- Replace `<zone>` with the zone you wish to launch the VSI in. We suggest using `us-south-1` for this how-to.
 
-See https://cloud.ibm.com/docs/vpc?topic=vpc-profiles&interface=ui for information on selecting a profile.
+Replace ``<vpc-id>``, ``<zone>``, ``<image-id>`` and ``<ssh-key-id>`` with the information gathered above. If you need help deciding on the instance-type refer to `IBM's documentation on instance profiles`_.
 
-A filled out example command would look like this:
+A complete example command would look like this:
 
 .. code-block:: bash
 
    ibmcloud is instance-create TODO: fill in the rest of the command
```

Comment:
@a-dubs could you fill this in? We missed it in our last round of reviews.

Reply:
AH RIGHT OOPS

the image id used for this command (r006-3a44e4ee-9c9f-4693-98ae-fced7a46ffce) is for Ubuntu 24.04 LTS.
```
ibmcloud is instance-create MyUbuntuInstance \ 
    r006-613dd231-0123-4567-8901-1de1ce8116fa us-south-1 bx2-2x8 r006-3a44e4ee-9c9f-4693-98ae-fced7a46ffce \
    --keys r006-b1fd6d2c-abcd-ef12-3456-4f44d2e16aac 
```